spec/requests/api/v1/questions_spec.rb

```diff
@@ -17,6 +17,9 @@
     let(:question_2) do
       FactoryBot.create(:question, user_id: user_2.id)
```

Comment:
Add `private: false` since the `Question` factory is set to initialize a random boolean value.

Reply:
https://github.com/SpicyCookies/rails-interview-project/pull/3/commits/a2ac18f7abdc02074b72af437ec97cc7e1a770fb